core/src/main/java/org/geysermc/geyser/entity/type/Entity.java

```diff
@@ -216,19 +216,41 @@ public void moveRelative(double relX, double relY, double relZ, float yaw, float
     }
 
     public void moveRelative(double relX, double relY, double relZ, float yaw, float pitch, float headYaw, boolean isOnGround) {
-        setYaw(yaw);
```

Comment:
Why did you remove these? They save the position of the entity for us in code.

Reply:
isn't this.yaw and setYaw not the same, and it is saved during the comparisation (just little re-ordered, to effiencently use the stack, as it is already loaded onto the stack in bytecode)

Reply:
I see it now.